File: static/code/sample/tethered/gdl-hislider-b.py
# In this sample:
# Update Slide value every second

import time
from DUELink.DUELinkController import DUELinkController

availablePort = DUELinkController.GetConnectionPort()
duelink = DUELinkController(availablePort)

# method
def Slide():
    return duelink.Engine.ExecuteCommand("Slide()")

while True:
    print(f"Value: {Slide()}")
    time.sleep(1)  # Sleep for 1 second

File: static/code/sample/tethered/gdl-hislider-b.cs
// In this sample:
// Update Slide value every second

using System;
using GHIElectronics.DUELink;

var availablePort = DUELinkController.GetConnectionPort();
var duelink = new DUELinkController(availablePort);

float Slide() {    
    return duelink.Engine.ExecuteCommand("Slide()");
}

while (true) {    
    Console.WriteLine($"Value: {Slide()}");
    Thread.Sleep(1000);
}

